scripts/create_nested_json.py

```diff
@@ -0,0 +1,42 @@
+from pandas import read_csv
+import os
+import json
+
+DISHES_CSV_PATH = "../images.adb.csv"
+INGREDIENTS_CSV_PATH = "../ingredients.csv"
+
+print(os.getcwd())
+dishes = read_csv(DISHES_CSV_PATH)
+ingredients = read_csv(INGREDIENTS_CSV_PATH)
+
+json_rep = json.loads(dishes.to_json(orient="records"))
+
+def remove_extra(selected):
+    for ing in selected:
+        extra = ["id",
+                "name",
+                "filename"]
+        [ing.pop(k) for k in extra if k in ing]
+        for k in ing.copy():
+            if not ing[k]:
+                ing.pop(k)
+    return selected.copy()
+
+
+if __name__ == "__main__":
```

Comment:
Not sure I follow the need for this script --> more specifically - what is this method?